clusterDeployer.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, cc, ai, args, secrets_path: str):
         self._iso_path = "/root/iso"
         os.makedirs(self._iso_path, exist_ok=True)
         self._extra_config = {}
+        self._sno = False
```

Comment:
Let's create a function that returns `len(self._cc["masters"]) == 1 and len(self._cc["workers"]) == 0`

Reply:
Will do.